main_page.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./css/main_page.css">
+    <link rel="stylesheet" href="./css/fancybox.css">
+    <title>main_page</title>
+</head>
+
+<body>
+    <div class="page-one">
+        <div class="nav">
+            <div class="nav__add"><a href="#" data-fancybox data-src="#hidden-content">Добавить</a></div>
+            <div class="hidden-content" id="hidden-content">
+                <form class="modal-content" action="#">
+                    <input type="text" placeholder="Название">
+                    <input type="text" placeholder="Текст">
+                    <input type="text" list="list">
+                    <datalist id="list">
+                        <option value="Все">
+                        <option value="Мода">
+                        <option value="Не мода">
+                    </datalist>
+                    <input type="submit" value="Добавить">
+                </form>
+            </div>
+            <div class="nav__exit"><a href="#">Выход</a></div>
+        </div>
+
+        <div class="categories">
+            <div class="categories__all"><a href="#">Все</a></div>
+        </div>
+
+        <div class="blog">
+            <div class="blog__name"><a href="#">Название статьи</a></div>
+            <div class="blog__categories">Все</div>
+            <div class="blog__data">24.01.2023</div>
+            <div class="blog__description">описание статьи описание статьи описание статьи</div>
+        </div>
```

Comment:
how are you going to place plural items?

Reply:
do I need to make a list?